custom_connectors/oauth2/bim_360.rb

```diff
@@ -3180,10 +2130,11 @@
         ]
       end,
       poll: lambda do |_connection, input, closure|
+        hub_id = closure&.[]('hub_id') || input['hub_id']
         project_id = closure&.[]('project_id') || input['project_id']
         container_id = closure&.[]('container_id') ||
                        get("/project/v1/hubs/#{input['hub_id']}" \
-                           "/projects/#{project_id}")&.
+                           "/projects/#{input['project_id']}")&.
```

Comment:
you can use `project_id ` field instead of `input['project_id']`

Reply:
same comment as above. this ensures we're looking at the folder specified in the input instead of the cached value.